pkg/job/job.go

```diff
@@ -302,6 +303,10 @@ func (k *kubeJob) GenerateISO(ctx context.Context, cluster common.Cluster, jobNa
 func (k *kubeJob) createKubeconfigJob(cluster *common.Cluster, jobName string, cfg []byte, encodedDhcpFileContents string) *batch.Job {
 	id := cluster.ID
 	ignitionGeneratorImage := k.Config.IgnitionGenerator
+	var pullPolicy core.PullPolicy = "Always"
+	if k.Config.SubsystemRun {
```

Comment:
why not making the policy as a flag?

Reply:
Why not have the `always` flag?
Since (see my comment on the default name of the image) will never be in external repo, `always` should work, no?

Reply:
@ronniel1 always flag means that kuberenetes will try to pull the image from external repo every time, that's why we need to set never on subsystem